frontend/src/app/patients/UploadPatients.tsx

```diff
@@ -0,0 +1,319 @@
+                To upload patients in bulk, spreadsheets need to match
+                SimpleReport data requirements. Follow the Patient bulk upload
+                guide closely to format your CSV before uploading it below.
+              </p>
+            </div>
+            <Button className={"margin-right-105"}>
+              View patient bulk upload guide
+            </Button>
+            <Button variant={"outline"}>Download spreadsheet template</Button>
+          </div>
+          <div
+            className={
+              "margin-top-7 margin-bottom-1 font-sans-lg text-light line-height-225"
+            }
+          >
+            2. Would you like to import these patients to one facility OR all
+            facilities?
+          </div>
+          <div className={"margin-left-3"}>
+            <div className={"maxw-710 line-height-sans-4"}>
+              If you plan to test patients at more than one facility, we
+              recommend adding them to all facilities. You can't select multiple
+              facilities individually.
+            </div>
+            <RadioGroup
+              wrapperClassName="margin-top-2"
+              inputClassName={"usa-radio__input--tile"}
+              name="facilitySector"
+              legend="Select facility"
+              legendSrOnly
+              buttons={[
+                {
+                  value: "oneFacility",
+                  label: "One facility",
+                },
+                {
+                  value: "allFacility",
+                  label: "All facilities",
+                },
+              ]}
+              selectedRadio={facilityAmount}
+              onChange={setFacilityAmount}
+              variant="horizontal"
+            />
+            {facilityAmount === "oneFacility" && (
+              <div className={"margin-top-205"}>
+                <div>Which facility?</div>
+                <Dropdown
+                  aria-label={"Select facility"}
+                  selectedValue={facility.id}
+                  onChange={onFacilitySelect}
+                  className={"grid-col-4"}
+                  options={facilities.map(({ name, id }) => ({
+                    label: name,
+                    value: id,
+                  }))}
+                />
+              </div>
+            )}
+          </div>
+          <div
+            className={
+              "margin-top-7 margin-bottom-1 font-sans-lg text-light line-height-225"
+            }
+          >
+            3. Upload your spreadsheet
+            {facilityAmount === "oneFacility" && " for " + facility.name}.
+          </div>
+          <div className={"margin-left-3"}>
+            <div className={"maxw-710 line-height-sans-4"}>
+              The spreadsheet may take 10 or more minutes to upload. You do not
+              need to stay on this page. We'll email you if you need to fix any
+              errors or when the upload is complete.
```

Comment:
@lauraykerr or @kenieh, feedback on the copy here: It's slightly incorrect, if there are errors that need to be fixed that will be apparent to the user immediately. We'll only email them once the upload is complete.